tests/unit/amo/components/TestUserProfilePictureEdit.js

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react';
+
+import UserProfileEditPicture, {
+  UserProfileEditPictureBase,
+} from 'amo/components/UserProfileEditPicture';
+import { getCurrentUser } from 'amo/reducers/users';
+import UserAvatar from 'ui/components/UserAvatar';
+import {
+  dispatchSignInActions,
+} from 'tests/unit/amo/helpers';
+import {
+  fakeI18n,
+  shallowUntilTarget,
+} from 'tests/unit/helpers';
+
+
+describe(__filename, () => {
+  const render = ({ i18n = fakeI18n(), ...props } = {}) => {
+    return shallowUntilTarget(
+      <UserProfileEditPicture i18n={i18n} name="input-name" {...props} />,
```

Comment:
This component has some mandatory props that are not being passed. I guess the tests don't care, and I honestly don't recall if we usually worry about that or not, but I wonder if we should have default props for everything that is mandatory?

Reply:
That would make sense, thanks!